PyZ3950/c2query.py

```diff
@@ -1,10 +1,6 @@
-
 #!/usr/local/bin/python2.3
+from io import StringIO
```

Comment:
In Python 2 `io.StringIO` != `StringIO.StringIO`
* https://stackoverflow.com/questions/3410309/what-is-the-difference-between-stringio-and-io-stringio-in-python2-7
```suggestion
try:
    from StringIO import StringIO  # Python 2
except ImportError:
    from io import StringIO  # Python 3
```